releng/helm-charts/xsk-landscape/templates/NOTES.txt

```diff
@@ -1,9 +1,7 @@
-{{- if .Values.namespace.name }}
+{{- if .Release.Name }}
 1.
-Created namespace {{.Values.namespace.name}}
-{{- end }}
+Created namespace {{ .Release.Name }}
```

Comment:
Isn't this `{{ Release.Namespace }}`?

Reply:
Yes it is and aligning the parameters with the other charts is better. I will change.